### PR TITLE
sysbench: add page

### PR DESCRIPTION
**Pull request by allesklardy**
- [y] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [y] The page(s) have at most 8 examples.
- [y] The page description(s) have links to documentation or a homepage.
- [y] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known): 1.0.20**

### CODE REVIEW

File: pages/common/sysbench.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# sysbench
+
+> Benchmark your System's cpu, io and memory.
```
Comment (navarroaxel) [inline pages/common/sysbench.md:3]:
```suggestion
> Benchmark your System's CPU, IO and memory.
```

File: pages/common/sysbench.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+> More information: <https://github.com/akopytov/sysbench/>.
```
Comment (navarroaxel) [inline pages/common/sysbench.md:4]:
```suggestion
> More information: <https://github.com/akopytov/sysbench>.
```

File: pages/common/sysbench.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`sysbench memory run`
+
+- Prepare a fileio read benchmark:
+
+`sysbench fileio prepare`
+
+- Run a fileio benchmark {seqwr, seqrewr, seqrd, rndrd, rndwr, rndrw}:
+
+`sysbench --file-test-mode={{test_mode}} fileio run`
```
Comment (navarroaxel) [inline pages/common/sysbench.md:24]: If these are the modes should be here
```suggestion
`sysbench --file-test-mode={{rndrd|rndrw|rndwr|seqrd|seqrewr|seqwr}} fileio run`
```

File: pages/common/sysbench.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+- Run a memory benchmark with 1 thread for 10 seconds:
+
+`sysbench memory run`
+
+- Prepare a fileio read benchmark:
+
+`sysbench fileio prepare`
+
+- Run a fileio benchmark {seqwr, seqrewr, seqrd, rndrd, rndwr, rndrw}:
```
Comment (navarroaxel) [inline pages/common/sysbench.md:22]:
```suggestion
- Run a filesystem-level benchmark:
```

File: pages/common/sysbench.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
+# sysbench
+
+> Benchmark your System's cpu, io and memory.
+> More information: <https://github.com/akopytov/sysbench/>.
+
+- Run a cpu benchmark with 1 thread for 10 seconds:
```
Comment (navarroaxel) [inline pages/common/sysbench.md:6]:
```suggestion
- Run a CPU benchmark with 1 thread for 10 seconds:
```

File: pages/common/sysbench.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+`sysbench cpu run`
+
+- Run a cpu benchmark with multiple threads for a specified time:
```
Comment (navarroaxel) [inline pages/common/sysbench.md:10]:
```suggestion
- Run a CPU benchmark with multiple threads for a specified time:
```

File: pages/common/sysbench.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+`sysbench --threads={{number_of_threads}} --time={{seconds}}`
+
+- Run a memory benchmark with 1 thread for 10 seconds:
+
+`sysbench memory run`
+
+- Prepare a fileio read benchmark:
```
Comment (navarroaxel) [inline pages/common/sysbench.md:18]:
```suggestion
- Prepare a filesystem-level read benchmark:
```

**Review by navarroaxel — APPROVED** [review]

File: pages/common/sysbench.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
+# sysbench
+
+> Benchmark your System's CPU, IO and memory.
```
Comment (CleanMachine1) [inline pages/common/sysbench.md:3]:
```suggestion
> Benchmark a System's CPU, IO and memory.
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**